src/backend/src/utils/users.utils.ts

```diff
@@ -36,6 +36,20 @@ export const getUsers = async (userIds: number[]): Promise<User[]> => {
   return users;
 };
 
+/**
+ * Produce a array of primsa formated userIds, given the array of User
+ * @param userIds the userIds to get as users
+ * @returns userIds in prisma format
+ */
+export const getUserPrismaIds = (users: User[]) => {
```

Comment:
this is more of a tool for querying than a `getUserPrismaIds`, I'd say, getPrismaQueryUserIds